apps/screloc/main.cpp

```diff
@@ -403,7 +434,19 @@ class Reloc : public Client::Application {
 			}
 		}
 
-		void addObject(const std::string &parentID, Object *obj) {
+		void printUsage() const override {
+			cout << "Usage:"  << endl << "  " << name() << " [options]" << endl << endl
+			     << "Relocate origins" << endl;
+
+			Client::Application::printUsage();
+
+			cout << "Examples:" << endl;
+			cout << "Offline processing of all origins stored in XML file" << endl
+			     << "  " << name() << " -d localhost --ep origins.xml"
+			     << endl << endl;
+		}
+
```

Comment:
I would put this change in the same commit as the reformatting one

Reply:
will do so

Reply:
done